server/api/project_apis.py

```diff
@@ -248,9 +253,12 @@ def get(self):
             search_dto.campaign_tag = request.args.get('campaignTag')
             search_dto.page = int(request.args.get('page')) if request.args.get('page') else 1
             search_dto.text_search = request.args.get('textSearch')
-            if tm.authenticated_user_id and \
-                    UserService.is_user_a_project_manager(tm.authenticated_user_id):
-                search_dto.is_project_manager = True
+            try:
```

Comment:
Seems sensible enough to me.  I would add a comment here to explain what's happening as the search functionality is getting v. complex so might help new comers understand the intent.